src/libraries/System.Threading.Channels/ref/System.Threading.Channels.netcoreapp.cs

```diff
@@ -11,8 +11,4 @@ public partial class ChannelClosedException : System.InvalidOperationException
     {
         protected ChannelClosedException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
I'm not sure why this is limited to netcoreapp.  Can this whole file just be renamed to System.Threading.Channels.netstandard21.cs, rather than splitting it?

Reply:
combined back and renamed to System.Threading.Channels.netstandard21.cs